README.md

```diff
@@ -507,6 +509,526 @@ source scripts/makeAllPlots.sh figures/ResponseUniformityMaps
 
 this will then execute genericPlotter taking each `*.cfg` file in the `figures/ResponseUniformityMaps` directory.
 
+### 3.b.ii.I Helper Script - Parsing Excel Files to genericPlotter 
+
+- **Main script:** Produce\_Config\_File.py
+- **Additional scripts:** QC3.py, QC4\_HV.py, QC4\_SS.py, QC5\_Gain.py, QC5\_Rate.py
```

Comment:
As stated [here](https://github.com/bdorney/CMS_GEM_Analysis_Framework/pull/12/files#diff-26fd799ea07494916e9da9b91b2aac64R713) these should be **separate** subsections.  One for an arbitrary excel file and one for the `QCX*.py` files.  Please change the title accordingly.

Reply:
Additionally don't refer to `Produce_Config_File.py` as "the main script" there isn't a single "main script" in the framework, there are over 20 scripts.  Refer to it by name exactly, e.g. `Produce_Config_File.py`

Reply:
done..

Reply:
done